src/Core/Content/Media/Extension/ResolveRemoteThumbnailUrlExtension.php

```diff
@@ -21,11 +23,18 @@ final class ResolveRemoteThumbnailUrlExtension extends Extension
      */
     public function __construct(
         public string $mediaUrl,
+        /**
+         * @deprecated is resolved in the extension function
```

Comment:
please add proper deprecation tags. you should find multiple examples in the code :blush: